### PR TITLE
eMIS Removal - Test User Data

### PR DESCRIPTION
**Pull request by rjohnson2011**
## Summary
Originally eMIS removal was part of one PR, however there were a number of E2E failures and the eMIS removal is now being broken apart into different PRs in an attempt to isolate where the errors are occurring and address those on an individual basis.

## Related issue(s)

- Investigation Zenhub ticket - https://app.zenhub.com/workspaces/platform-product-team-633af4074573d06c3cda142a/issues/gh/department-of-veterans-affairs/va.gov-team/72209
- Removal of test user data ticket - https://app.zenhub.com/workspaces/platform-product-team-633af4074573d06c3cda142a/issues/gh/department-of-veterans-affairs/va.gov-team/75089

## Testing done

- Passing all specs & E2E tests

## What areas of the site does it impact?

- Test user data regarding eMIS

### CODE REVIEW

File: src/applications/disability-benefits/2346/tests/data/users/eddieUserData.json
Comment (rmtolmach) [inline src/applications/disability-benefits/2346/tests/data/users/eddieUserData.json:1]: What are all of these `.json` files? They seem like some kind of information that was recorded at some point. I'm wondering if we need to re-record it... if that's a thing?? 🤷

Comment (rjohnson2011) [reply src/applications/disability-benefits/2346/tests/data/users/eddieUserData.json:1]: Would love to  please get Frontend CoP's opinion on this @CBonade

**Review by rmessina1010 — APPROVED** [review]
LGTM

File: src/applications/disability-benefits/2346/tests/data/users/eddieUserData.json
Comment (LindseySaari) [reply src/applications/disability-benefits/2346/tests/data/users/eddieUserData.json:1]: Yeah I think we should follow up on @rmtolmach's comment.  It seems like it has _something_ to do with mockdata and betamocks. Ideally, if we could remove the EMIS reference, that would be best.

Comment (LindseySaari) [reply src/applications/disability-benefits/2346/tests/data/users/eddieUserData.json:1]: Here is [the original PR](https://github.com/department-of-veterans-affairs/vets-website/pull/13167) for some of the mockdata. Looks like it's part of the Cypress e2e testing.

Comment (rmtolmach) [reply src/applications/disability-benefits/2346/tests/data/users/eddieUserData.json:1]: > What are all of these .json files?

answering my own question 😂 These are responses that can seen in the browser developer tools (Network -> user -> Response) after logging into staging as the user listed in the file (Eddie, in this case).

**Comment by rmtolmach** [comment]
@rjohnson2011 my work in https://github.com/department-of-veterans-affairs/vets-website/pull/27851 will override these changes.

**Comment by rjohnson2011** [comment]
Closing this due to @rmtolmach 's PR.

**Pull request closed without merging**